Makefile

```diff
@@ -19,6 +19,8 @@ EOL_IMG ?= docker.io/library/alpine:3.1
 BUSYBOX_BASE_IMG ?= busybox:1.36.0
 NON_VULNERABLE_IMG ?= ghcr.io/azure/non-vulnerable:latest
 E2E_TESTS ?= $(shell find ./test/e2e/tests/ -mindepth 1 -type d)
+API_VERSIONS ?= $(shell find ./api/ -mindepth 1 -maxdepth 1 -type d -not -name unversioned -print0 | tr '\0' ',' | sed -E 's#,$$##')
+
```

Comment:
this doesn't seem to compile, I think we need to escape the #: 's\#,$$\#\#'

Reply:
i am okay to just adding paths manually for now since this is not going to get updated frequently

Reply:
I listed them manually for now